### DISCUSSION TITLE
Error converting value 3/26/2022 2:04:05 PM to type 'FikaAmazonAPI.AmazonSpApiSDK.Models.FulfillmentInbound.TimeStampStringType'. Path

### DISCUSSION THREAD

**Discussion by IncredibleX** [abuzuhri/Amazon-SP-API-CSharp#151] [Q&A]
Hi,

got the following Error: Newtonsoft.Json.JsonSerializationException
  HResult=0x80131500
  Nachricht = Error converting value 3/26/2022 2:04:05 PM to type 'FikaAmazonAPI.AmazonSpApiSDK.Models.FulfillmentInbound.TimeStampStringType'. Path 'payload.TransportContent.TransportDetails.PartneredSmallParcelData.PartneredEstimate.VoidDeadline', line 62, position 91.
  Quelle = Newtonsoft.Json
 
As workaround i changed VoidDeadline to string type.

**Comment by abuzuhri** [comment]
can you share the code give this error ?

**Comment by IncredibleX** [comment]
Hi,

the only thing i did is to change VoidDeadline to String:

Line 72 in your code: public string VoidDeadline { get; set; }


Perhaps it is a date/time format problem - solution could be to set another country via Globalization.
I used the quick and dirty solution. ;-)



Am 22. Apr. 2022, 06:13 +0200 schrieb Tareq Abuzuhri ***@***.***>:
